mir_eval/sonify.py

```diff
@@ -103,11 +103,18 @@ def time_frequency(gram, frequencies, times, fs, function=np.sin, length=None,
     if length is None:
         length = int(times[-1, 1] * fs)
 
-    times, _ = util.adjust_intervals(times, t_max=length)
-
-    # Truncate times so that the shape matches gram
-    n_times = gram.shape[1]
-    times = times[:n_times]
+    last_time_in_secs = float(length) / fs
+    times, _ = util.adjust_intervals(times, t_max=last_time_in_secs)
```

Comment:
Good catch on this!  Classic dimensional analysis error on my part, using sample indices in place of time measurements.

While trying to unpack what happened here, I noticed that the docstring isn't as explicit as it could be that `times` should be in units of seconds.  Could you add some clarifying text there?

Reply:
Added documentation string updates.